lib/pseudolocalization.rb

```diff
@@ -5,11 +5,12 @@ module Pseudolocalization
   module I18n
     class Backend
       attr_reader :original_backend
-      attr_accessor :ignores
+      attr_accessor :ignores, :only_locales
 
       def initialize(original_backend)
         @original_backend = original_backend
         @ignores = []
+        @only_locales = []
```

Comment:
Instead of defaulting to an empty array, and having custom logic to bypass the feature if the array is empty, would it be possible to instantiate this array to `I18n.available_locales`? Doing so would allow us to transform `if locale_ignored?(locale)` info `if pseudolocalized_locale?(locale)`, and simply have it check if the locale is part of the array. An added benefit would be the ability to set that array to empty, and completely bypass pseudolocalization for all locales.